website_portal_sale/README.rst

```diff
@@ -0,0 +1,45 @@
+========================
+Website Portal for Sales
+========================
+
+Backport (with adjustments) of the ``website_portal_sale`` module of Odoo 9.
+
+This module add the user's sales documents in the frontend portal.
+Customers will be able to see the list and state of their quotations, sales
+and invoices.
+
```

Comment:
Instructions should say that you need to activate the portal and invite a customer to it, and how to do it.